app/views/catalog/_show_collection.html.erb

```diff
@@ -38,9 +46,11 @@
       <div class='tab-pane' id='contents' role='tabpanel'>
         <%= render 'collection_contents' %>
       </div>
-      <div class='tab-pane' id='online-content' role='tabpanel'>
-        <%= render partial: 'collection_online_contents', locals: { document: document }  %>
-      </div>
+      <% if document.online_content? %>
+        <div class='tab-pane' id='online-content' role='tabpanel'>
+          <%# TODO: render children with online content in a search result style view %>
```

Comment:
@gordonleacock Can we retain this partial until someone picks off #927? Removing it now leaves the `Online content` tab empty. It would probably be easier for whoever works on #927 to iterate from that list of links.

Reply:
I added these partials back.